compiler/rustc_mir/src/dataflow/framework/engine.rs

```diff
@@ -285,12 +298,17 @@ where
         None if tcx.sess.opts.debugging_opts.dump_mir_dataflow
             && dump_enabled(tcx, A::NAME, def_id) =>
         {
+            // FIXME: Use some variant of `pretty::dump_path` for this
             let mut path = PathBuf::from(&tcx.sess.opts.debugging_opts.dump_mir_dir);
 
+            let crate_name = tcx.crate_name(def_id.krate);
             let item_name = ty::print::with_forced_impl_filename_line(|| {
                 tcx.def_path(def_id).to_filename_friendly_no_crate()
             });
-            path.push(format!("rustc.{}.{}.dot", item_name, A::NAME));
+
+            let pass_name = pass_name.map(|s| format!(".{}", s)).unwrap_or_default();
+
+            path.push(format!("{}.{}.{}{}.dot", crate_name, item_name, A::NAME, pass_name));
```

Comment:
So these changes change the output file to

`myowncrate.itemname....`

from

`rustc.itemname...`

I guess this is intended?

Reply:
Yeah, `pretty::dump_path` was updated to include the crate name, so nothing else in the `mir_dump` directory uses `rustc`. However, I don't have a `MirSource`, and the format is not quite right, so this was easier.